hyper-boring/tests/old.rs

```diff
@@ -1,12 +1,12 @@
-use super::*;
```

Comment:
We should probably name this `v0.rs` and name `hyper1.rs` `v1.rs` to be consistent with the implementation filse